profile.html

```diff
@@ -4,15 +4,35 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
+    <title>Quiz-App | Profile</title>
   </head>
   <body>
-    <nav>
-      <a href="./index.html">Homepage</a>
-      <a href="./bookmark.html">Bookmark</a>
-      <a href="./input.html">Input</a>
-      <a href="./profile.html">Profile</a>
-    </nav>
-    <p>Profile</p>
+    <header>
+      <h1>Profile</h1>
+    </header>
+    <main>
+      <img src="#" alt="Profile image" />
+      <h2>Firstname Lastname</h2>
+      <p>
+        Lorem ipsum dolor, sit amet consectetur adipisicing elit. Fuga
+        perferendis tempora voluptatem blanditiis voluptate illum placeat amet
+        excepturi ut dolor soluta vero, est at iste temporibus nemo porro beatae
+        sed!
+      </p>
+      <h3>Skills</h3>
+      <div>Tag 1</div>
```

Comment:
the same here with the unordered list. try to avoid divs and for little tags divs are the last possible option you should use for this

Reply:
Or to put it differently:
"The `<ul>` element is for grouping a collection of items that do not have a numerical ordering, and their order in the list is meaningless." [(MDN)](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul#usage_notes)
I would say something like "If several words / phrases belong together on the same level, but a `p` or whatever text content element does not really fit, think about a list."